namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-contacts-dev/resources/main.tf

```diff
@@ -0,0 +1,49 @@
+terraform {
+  backend "s3" {
+  }
+}
+
+provider "aws" {
+  region = "eu-west-2"
+
+  default_tags {
+    tags = {
+      source-code   = "github.com/ministryofjustice/cloud-platform-environments"
+      slack-channel = var.slack_channel
+      GithubTeam = "hmpps-move-and-improve"
+    }
+  }
+}
```

Comment:
This this section is additional to requirements? So london and ireland required - eu-west-1 and eu-west-2 - we have 2 x eu-west-1 definitions.

Reply:
Oh, looks like its duplicated in the 'hmpps-challenge-support-intervention-plan-dev' folder where I have copied it from , removed now form this file.